### ISSUE TITLE
Terminal unexpectedly opens when using "find references"

### ISSUE THREAD

**Issue by OliverJAsh** [jest-community/vscode-jest#957]
### Environment

1. `vscode-jest version`: v4.6.0
2. `node -v`: v18.12.1
3. `npm -v` or `yarn --version`: 1.22.19
4. `npm ls jest` or `npm ls react-scripts` (if you haven’t ejected): jest@29.0.3
5. your vscode-jest settings if customized:
    - jest.jestCommandLine? `just unit`
    - jest.autoRun? `"off"`
    - anything else that you think might be relevant? [fill]
6. Operating system: macOS 13.0.1

### Prerequisite
- are you able to run jest test from the command line? Yes
- how do you run your tests from the command line? (for example: `npm run test` or `node_modules/.bin/jest`) `just unit`

### Steps to Reproduce

1. Go to a source file
2. Use find references on a function to find a reference inside a test file

### Relevant Debug Info

N/A

### Expected Behavior

The terminal does not open.

### Actual Behavior

The terminal opens with the following output:

```
[unsplash-web]: resolving children for /Users/oliverash/Development/unsplash/unsplash-web/shared/helpers/urls/__tests__/url.test.ts


> Test run finished at 30/11/2022, 15:43:10 <
```

https://user-images.githubusercontent.com/921609/204841821-2edad48b-2cfa-4b46-ad0f-4388d50c5c72.mov

I can workaround this behaviour using the setting `"jest.showTerminalOnLaunch": false`, but I don't think this is a solution. I still want the terminal to launch when I run tests, just not when I'm using find references.

**Comment by connectdotz** [comment]
do you mind giving [v5 pre-release](https://github.com/jest-community/vscode-jest/releases/tag/v5.0.2) a try, we have changed the output mechanism.

**Comment by SpencerKaiser** [comment]
@connectdotz can you share some feedback on what you're changing? I'm on the verge of uninstalling because this keeps hijacking my terminal when I start. Now it seems that every single failing runner will ALSO hijack focus when something goes wrong and it's driving me insane.

**Comment by connectdotz** [comment]
hm....  are you on 5.1? You could close the terminal and it should not be re-opened unless there is a "fatal" error (note, not test errors)... If you are still experiencing this in 5.1, can you do a screencast again so I can see what is happening?

**Comment by SpencerKaiser** [comment]
Yes I'm on 5.1 but it's 100% the fatal errors... I'm trying to fix a build error and every time my build terminal runs, it causes tests to re-run, the runner has a fatal error and then the terminal gets auto-focused to the test one, which is _exceptionally_ frustrating when I'm already frustrated by the build error 😅 

IMO there should be a `disableAutoTerminalFocus` flag that should just permanently disable that feature on start and on fatal errors. There's a chance that's all not in yall's control though...

Thanks for the quick feedback, hopefully this context is helpful!

**Comment by connectdotz** [comment]
I see... We do plan to release a new setting `jest.autoRevealOutput` in the next pre-release. You can try this [pre-pre-release](https://github.com/jest-community/vscode-jest/issues/973#issuecomment-1374932071) to stop the terminal from auto-reveal, if you want. Just set `"jest.autoRevealOutput": "off"`, and delete it when your project is back to normal.

**Comment by SpencerKaiser** [comment]
Awesome, looking forward to it! Thank you so much for the help and quick responses 🙂

**Comment by connectdotz** [comment]
Hi, the fix `jest.autoRevealOutput` is released in [5.2.0 pre-release](https://github.com/jest-community/vscode-jest/blob/master/release-notes/release-note-v5.x.md#v520-pre-release); feel free to give it a try and let us know if there is any issue.
